src/main/java/software/amazon/smithy/lsp/Main.java

```diff
@@ -48,17 +61,14 @@ public static void main(String[] args) {
         in = socket.getInputStream();
         out = socket.getOutputStream();
       }
-      SmithyLanguageServer server = new SmithyLanguageServer();
-      Launcher<LanguageClient> launcher = LSPLauncher.createServerLauncher(server, in, out);
-      LanguageClient client = launcher.getRemoteProxy();
 
-      server.connect(client);
-
-      launcher.startListening().get();
+      throw launch(in, out).get();
```

Comment:
That (combined with the catch) is weird looking, I'd rather it was an if(opt.isPresent) {throw } else {System.out.println(...) }

Reply:
Yeah, it looked stupid even when I wrote it, and then forgot. Will change